### ISSUE TITLE
How to Disable Default SSLStrip?

### ISSUE THREAD

**Issue by ngodai** [byt3bl33d3r/MITMf#358]
Hi everybody!
Can I disable default sslstrip?
Because when I used MITMf attack a victim, then victim download an exe file from https server. After that, I couldn't run this exe file. Then, I compared it with real exe file(which downloaded when not attack) all symbol 's' in https is missing.
I try to remove this code in file mitmf.py:

    from core.sslstrip.CookieCleaner import CookieCleaner
    from core.sslstrip.StrippingProxy import StrippingProxy
    from core.sslstrip.URLMonitor import URLMonitor
    URLMonitor.getInstance().setFaviconSpoofing(options.favicon)
    URLMonitor.getInstance().setCaching(options.preserve_cache)
    CookieCleaner.getInstance().setEnabled(options.killsessions)

    strippingFactory          = http.HTTPFactory(timeout=10)
    strippingFactory.protocol = StrippingProxy

    reactor.listenTCP(options.listen_port, strippingFactory)

    for plugin in plugins:
        if vars(options)[plugin.optname] is True:
            plugin.reactor(strippingFactory)

    print "|_ Sergio-Proxy v0.2.1 online"
    print "|_ SSLstrip v0.9 by Moxie Marlinspike online"

But I cannot access any website. 
Sorry, my English 's not good!

**Comment by n0rp3d** [comment]
You've got to be kidding me. You've been further even more decided to use even go need to do look more as anyone can. Can you really be far even as decided half as much to use go wish for that?

**Comment by byt3bl33d3r** [comment]
TL;DR no you cannot, the SSLStrip functionality is a core part of the framework.

 As for the exe downloading stuff not really sure what you mean.
